Sources/AWSLambdaRuntimeCore/LambdaContext.swift

```diff
@@ -138,15 +138,15 @@ extension Lambda {
             self.storage.allocator
         }
 
-        internal init(requestID: String,
-                      traceID: String,
-                      invokedFunctionARN: String,
-                      deadline: DispatchWallTime,
-                      cognitoIdentity: String? = nil,
-                      clientContext: String? = nil,
-                      logger: Logger,
-                      eventLoop: EventLoop,
-                      allocator: ByteBufferAllocator) {
+        public init(requestID: String,
```

Comment:
@tomerd We probably need to discuss this one! Maybe we want to add a `__forTest_Only()` helper method instead of the public initializer.